### DISCUSSION TITLE
Is it possible to send text?

### DISCUSSION THREAD

**Discussion by jstec** [sipsorcery-org/sipsorcery#927] [Q&A]
I know that audio and video communication is possible, Can I send text between peers too?
Thank you

**Comment by imtadashi** [comment]
Yes, refer this example. [Sipsorcery WebRTCGetStartedDataChannel](https://github.com/sipsorcery-org/sipsorcery/tree/master/examples/WebRTCExamples/WebRTCGetStartedDataChannel)
